### PR TITLE
logging flags

### PR DESCRIPTION
**Pull request by jseldess**
Added logging flags to docs. Does this looks correct and the right level of details? @bdarnell, @BramGruneir, @petermattis ?

HTML version here: http://cockroach-draft-docs.s3-website-us-east-1.amazonaws.com/cockroach-commands.html#logging-flags

### CODE REVIEW

File: start-a-node.md [hunk 1]
```diff
@@ -19,26 +19,17 @@ $ ./cockroach help start
...
+`--host` | The address advertised to other nodes and at which the node can be reached by other nodes and clients. This may be the node's internal hostname, internal ip address, external hostname, external ip address, etc. <br><br>**Default:** Advertise node's hostname and listen on all interfaces
 `--insecure` | Whether or not the cluster is secure (authentication and encrypted client/node and inter-node communication). If the cluster is secure, set the `--certs` flag but leave this flag out. If the cluster is insecure, set this flag.
 `--join` | The address for connecting the node to an existing cluster. When starting the first node, leave this flag out. When starting subsequent nodes, set this flag to the address of any existing node. Optionally, you can specify the addresses of multiple existing nodes as a comma-separated list. 
-`--port` | The port over which the node communicates to the rest of the cluster and clients communicate to the node. <br><br>**Default:** 26257
+`--port` | The port over which the node communicates to other nodes in the cluster and clients communicate to the node. <br><br>**Default:** 26257
```
Comment (petermattis) [inline start-a-node.md]: The node doesn't communicate using this port, but other nodes communicate to this node using this port. I think the whole idea of using communication is a bit fraught. Perhaps something like: `The port the node listens on for internal and client communication.`

Comment (jseldess) [inline start-a-node.md]: Revised the `--port` description in both start-a-node.md and stop-a-node.md. 

@petermattis, does the **Logging Flags** section look right?

Comment (petermattis) [inline start-a-node.md]: Yep. Looks good.

File: cockroach-commands.md [hunk 2]
```diff
@@ -23,6 +22,13 @@ Command | Usage
 
 ## Logging Flags
 
+By default, CockroachDB logs all messages and errors to files (see `--log-dir`) and copies errors with a severity of `ERROR` or higher to the standard error stream (see `--alsologtostderr`). 
+
 Flag | Description
 -----|------------
- | 
+`--alsologtostderr` | Copy log messages at or above this severity level to the standard error stream in addition to log files. Possible values: `INFO`, `WARNING`, `ERROR`, and `FATAL`. <br><br>**Default:** `ERROR`
```
Comment (bdarnell) [inline cockroach-commands.md]: Do these terms need to be in all caps? I'd prefer to use lowercase if that works (and if it doesn't, we should make it work)

Comment (petermattis) [inline cockroach-commands.md]: No, the `{INFO,WARNING,ERROR,FATAL}` terms do not need to be in all caps. Any capitalization is ok.

Comment (jseldess) [inline cockroach-commands.md]: Yep, lowercase works. I'll update that.

**Pull request merged**